src/modules/PopupMovies.js

```diff
@@ -0,0 +1,117 @@
+          return [];
+        }
+        return response;
+      } catch (error) {
+        return [];
+      }
+    };
+
+    showPopup = async (movie) => {
+      this.currentMoviesID = movie.id;
+      const projectModal = document.querySelector('.popup');
+      const body = document.querySelector('main');
+      projectModal.style.display = 'block';
+      const modalVisible = `
+                  <div class="popup-modal">
+                  <div class="image-close flex-center">
+                      <div class="modal-image">
+                          <img src="${movie.image.medium}" alt="">
+                          <h3 class="flex-center movie-title">${movie.name}</h3>
+                      </div>
+                      <div class="close"> <i class="fa-solid fa-xmark"></i></div>
+                  </div>
+                  <div class="flex-center description">
+                  <div class="start-end">
+                  <p> Premiered: <span>${movie.premiered}</span></p>
+                  <p>Ended: <span>${movie.ended}</span></p>
+                </div>
+                <div class="movie-for">
+                  <p>Genres: <span>${movie.genres}</span></p>
+                  <p>language: <span>${movie.language}</span></p>
+                </div>
+                  </div>
+                  <h4 class="flex-center"> comments(<span class ="count">0</span>)</h4>
+                  <div class="flex-center comment-modal">
+                      <div class="comment-container flex-center">
+                          
+                      </div>
+                  </div>
+                  <div class="flex-center type-comment-modal">
+                      <form class="form flex-center">
+                          <input type="text" name="name" id="name" placeholder="Your name">
+                          <textarea class="textarea" name="comment" id="comment" cols="30" rows="10"></textarea>
+                          <button class="commnet-btn">comment</button>
+                      </form>
+      
+                  </div>
+      
+              </div>`;
+      body.style.position = 'fixed';
+      projectModal.innerHTML = modalVisible;
+
+      /* event listener */
+      /* Add new comments */
+      const Submit = document.querySelector('.commnet-btn');
+      Submit.addEventListener('click', async (event) => {
+        event.preventDefault();
+        const userName = document.querySelector('#name');
+        const comment = document.querySelector('#comment');
+        await this.postComment(movie.id);
+        this.displayAllComment();
+        userName.value = '';
+        comment.value = '';
+      });
+
+      const closeModal = document.getElementsByClassName('close');
+      for (let i = 0; i < closeModal.length; i += 1) {
+        closeModal[i].addEventListener('click', () => {
+          projectModal.style.display = 'none';
+          body.style.position = 'relative';
+        });
+      }
```

Comment:
- [ ] I suggest you use `document.querySelector('.close')` instead of `document.getElementByClassName` so we won't need to use a loop
Here is how it should look (I tested the code, and it works fine)

```suggestion
      const closeModal = document.querySelector('.close')
      closeModal.addEventListener('click', () => {
        projectModal.style.display = 'none';
        body.style.position = 'relative';
      });
```

Reply:
@Trast00 alright sir, I will do that in the next pull request